apps/shooting-clubs/models/submission.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
+const _ = require('lodash');
 
 module.exports = data => {
   const response = {};
+  const activity = [
+    { activity: 'new', response: 'Application' },
+    { activity: 'renew', response: 'Renewal' },
+    { activity: 'vary', response: 'Vary' }
+  ];
 
   response.AuthorityType = 'Shooting clubs';
-  response.ApplicationType = data.activity === 'new' ? 'Application' : 'Renewal';
+  response.ApplicationType = data.activity ?
+    _.find(activity, { activity: data.activity }).response : 'Renewal';
```

Comment:
this is very hard to read and understand what's going on? consider using an if 🙂